AiTechEye/multidimensions
# Remove hard dependency on default

Does not change the current behavior in minetest game but makes all default specific code (including the dimension registrations) conditional on default being present and moves the dependency on default to optional_depends.

This is to make it possible to use the mutli dimensions api in other games than mtg-derivates.

Also fixes some whitespace messes and windows line endings.

**Testing:**
* Verify the mod still works like before in minetest_game 
* Verify the mod can be loaded in games that do not provide default (e.g. void game)
